tests/python/unittest/test_hybrid_script.py

```diff
@@ -680,13 +710,15 @@ def outer_product(a, b):
     ir = ir.body
     assert isinstance(ir, tvm.stmt.For)
     assert ir.loop_var.name == 'i.j.fused'
-    run_and_check(outer_product, [a, b], sch=sch, outs=[c])
+    func, ins, outs = run_and_check(outer_product, [a, b], sch=sch, outs=[c])
+    run_and_check(func, ins, outs=outs)
 
     # Test imperfect loop split
     sch = tvm.create_schedule(c.op)
     sch[c].split(c.op.axis[0], 3)
     ir = tvm.lower(sch, [a, b, c], simple_mode=True)
     run_and_check(outer_product, [a, b], sch=sch, outs=[c])
+    run_and_check(func, ins, outs=outs)
```

Comment:
Could you add a testcase for this flow, after hybrid building, save the dump into a file, and reload this file again, feed it to a normal build or just a normal pass.

Reply:
Good point. I should support load.